core/src/main/java/com/ritense/valtimo/domain/global/GlobalSettings.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.valtimo.domain.global;
+
+import org.hibernate.annotations.Type;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Map;
+import java.util.UUID;
+
+@Entity
+@Table(name = "global_settings")
+public class GlobalSettings {
+
+    @Id
+    @Column(name = "settings_id", nullable = false)
+    private UUID settingsId;
```

Comment:
I don't think we need a UUID. Instead, it might be useful to use a `category` as a primary key. The category could be something like 'ui', 'webui' or 'frontend' for frontend-settings. If we incorporate the key into the endpoint, we could add a bit of security based on the audience/client.